README.md

```diff
@@ -42,11 +42,24 @@ Change history can be found in [`HISTORY.md`](https://github.com/mongodb/node-mo
 
 ### Compatibility
 
-For version compatibility matrices, please refer to the following links:
+For server and runtime version compatibility matrices, please refer to the following links:
 
 - [MongoDB](https://www.mongodb.com/docs/drivers/node/current/compatibility/#mongodb-compatibility)
 - [NodeJS](https://www.mongodb.com/docs/drivers/node/current/compatibility/#language-compatibility)
 
+#### Component Support Matrix
+
+The following table describes add-on compponent version compatibility for the Node.js driver. Only packages with versions in these supported ranges are stable when used in combination.
+
+| Component                                                                            | `mongodb@3.x` | `mongodb@4.x`      | `mongodb@5.x`      | `mongodb@6.x` |
+| ------------------------------------------------------------------------------------ | ------------- | ------------------ | ------------------ | ------------- |
+| [bson](https://www.npmjs.com/package/bson)                                           | ^1.0.0        | ^2.0.0             | ^5.0.0             | ^6.0.0        |
+| [bson-ext](https://www.npmjs.com/package/bson-ext)                                   | ^1.0.0        | ^2.0.0             | N/A                | N/A           |
```

Comment:
Pretty certain the 4.x driver will only work with bson-ext 4+ and 3.x only works with 1.x/2.x

Reply:
let me double check that, I had a bunch of tabs open for the different dependencies and the corresponding package.json in the driver and it's possible by kerberos line got merged with the bson

Reply:
updated the table per slack discussion